app-frontend/src/app/pages/projects/edit/colormode/colormode.controller.js

```diff
@@ -20,49 +82,291 @@ export default class ProjectsEditColormode {
                         blueBand: correction.blueBand
                     };
                     this.correction = correction;
+                    if (!this.correction.mode) {
+                        this.correction.mode = 'multi';
+                    }
+                    this.$parent.fetchUnifiedComposites().then(() => {
+                        this.unifiedComposites =
+                            Object.assign(
+                                {},
+                                this.$parent.unifiedComposites,
+                                this.defaultColorModes
+                            );
+                        this.initProjectBuffer();
+                        this.isLoading = false;
```

Comment:
That is weird. Will investigate. Though, the loading icon for the map is contained in the map component so any `isLoading` here shouldn't be affecting that.